enhancements/machine-config/mco-cgroupsv2-support.md

```diff
@@ -201,19 +200,31 @@ The following jobs will be run against cgroup v2 periodically and with a minimum
 
 ### Upgrade / Downgrade Strategy
 
-Downgrading a cluster to an OpenShift version not containing cgroup v2 support
+- Downgrading a cluster to an OpenShift version not containing cgroup v2 support
 is unsupported.
+- Upgrading a cluster which is on cgroup v1 to a version >= 4.19 is blocked until it is migrated to cgroup v2
```

Comment:
describe how from a product development perspective and a customer experience perspective.  What does a customer mechanically have to do?